get-version.sh

```diff
@@ -1,7 +1,12 @@
-#!/bin/bash -ex
+#!/bin/bash -exu
 
-tag=$(git tag --points-at HEAD | grep -E '^[0-9]{8}(-[0-9]+)?$' |
-          sort --field-separator=- --key=1n,1 --key=2n,2 | tail -n1 || printf "")
+# Limit what branch can be to avoid weird regex in the call to sed
+branch=$(git rev-parse --abbrev-ref HEAD | grep -E '^[a-z0-9/-]+$' || printf "")
```

Comment:
what is the trailing || printf "" doing?

Reply:
Same as `|| true`

Reply:
Yes, you can do in different ways, it is just a way to ensure an empty branch variable if the grep command fails.